addons/account/i18n/account.pot

```diff
@@ -7367,6 +7367,11 @@ msgstr ""
 msgid "Journal group are used in reporting to display relevant data together."
 msgstr ""
 
+#. module: account
+#: model_terms:ir.actions.act_window,help:account.action_account_journal_group_list
+msgid "Journal groups are used in reporting to display relevant data together."
+msgstr ""
+
```

Comment:
```suggestion
msgid "Journal groups are used in reporting to display relevant data together."
msgstr ""
```
why duplication?

Reply:
@MiquelRForgeFlow That's our standard way of handling translation fixes in stable versions. Updating the existing term directly could break dbs relying on it, so we keep it and add a new, correct one.

Reply:
oh, right. thanks!